### DISCUSSION TITLE
Cfclient cannot detect crazyflie

### DISCUSSION THREAD

**Discussion by Keepgoing0925** [bitcraze/discussions#1717] [Q&A]
hi, I'm a new crazyflie user and I can't connect the drone in the cfclient. I'm on Windows and in particular, after inserting the crazyradio PA USB dongle and checking in device management that it is actually detected, I open the cfclient and turn on the crazyflie. By clicking on scan I am unable to find any crazyflies. Even if I connect the crazyflie to the USB I cannot configure it manually, as I want to underline that the drones I am using, given that they are in a laboratory, have been previously used by other people.
![8a46a4151666b9f50572249c6c7ea0b](https://github.com/user-attachments/assets/3b55dd89-864d-4bf3-876c-9396f843bae8)
![97e3cef58926b287a618e181625b318](https://github.com/user-attachments/assets/08c8cad6-b595-4da3-81cc-8e0cee378300)

https://github.com/user-attachments/assets/5e5c6b6e-56c8-4259-9735-336be04303ff

![d91eff118719e4ae5ea49ead034c202](https://github.com/user-attachments/assets/efeb23ad-7d0b-4820-9c1e-f36decac0e74)
![fad2b14cbd9a187577ba30637b7918a](https://github.com/user-attachments/assets/14ccb11e-1c74-4028-9d22-d5a0b578f542)
![fbdc5c0a43fcf3e67bcd6dea3144c8d](https://github.com/user-attachments/assets/2c572d14-d398-45c6-b92f-6a89ccf527a8)

**Comment by Urlaxle** [comment]
I had a similar issue yesterday where I could see that the dongle was detected by the system but not in the cfclient. 

It turned out that I just had to flash new firmware on my Crazyradio 2.0. It is pretty straightforward to do following the official tutorial: https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyradio-2-0/ 

If your Crazyflie have been used by others previously you would think the firmware already is updated but it may be worth a shot if it is still not working for you.

**Comment by ToveRumar** [comment]
Hi @Keepgoing0925 !
Thank you for supplying us with such detailed information about your issue!

Have you tried installing the Windows drivers as described here; https://www.bitcraze.io/documentation/repository/crazyradio-firmware/master/building/usbwindows/? 

If this does not work I would suggest trying the suggestion by @Urlaxle above to make sure you have working  and "up to date" FW on the radio.
